src/Action/DashboardAction.php

```diff
@@ -16,11 +16,11 @@
 use Sonata\AdminBundle\Admin\BreadcrumbsBuilderInterface;
 use Sonata\AdminBundle\Admin\Pool;
 use Sonata\AdminBundle\Templating\TemplateRegistryInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-final class DashboardAction extends Controller
+final class DashboardAction extends AbstractController
```

Comment:
A better goal IMO would be to completely remove the `extends AbstractController` part, and use dependency injection.

Reply:
See https://symfony.com/doc/current/controller/service.html for more information.